core/admin.py

```diff
@@ -0,0 +1,26 @@
+"""core admin panel"""
+
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+
+from core.models import User, Librarian
+
+
+@admin.register(Librarian)
+class LibrarianAdmin(admin.ModelAdmin):
+    """Registers Librarian model to admin panel"""
+
+
+@admin.register(User)
+class UserAdmin(BaseUserAdmin):
```

Comment:
register user should be using Django REST

Reply:
Can you please elaborate on this?

Reply:
sorry i thought it was for signup. my bad